app/src/main/java/de/tum/in/tumcampusapp/component/ui/transportation/DepartureView.kt

```diff
@@ -0,0 +1,185 @@
+package de.tum.`in`.tumcampusapp.component.ui.transportation
+
+import android.animation.ValueAnimator
+import android.content.Context
+import android.content.res.ColorStateList
+import android.graphics.Color
+import android.os.Handler
+import android.view.Gravity
+import android.view.LayoutInflater
+import android.view.View
+import android.view.animation.AccelerateInterpolator
+import android.view.animation.AnimationUtils
+import android.widget.LinearLayout
+import android.widget.TextSwitcher
+import android.widget.TextView
+import de.tum.`in`.tumcampusapp.R
+import org.joda.time.DateTime
+import org.joda.time.Seconds
+import java.util.*
+
+/**
+ * Custom view that shows a departure.
+ * Holds an icon of the subway public transfer line, the line name and an animated
+ * automatically down counting departure time
+ */
+class DepartureView
+/**
```

Comment:
I think we can get rid of this comment if you rename `big` to something more expressive, e.g. `useCompactView`.